### ISSUE TITLE
Trying to override verbose_name to magically export a language aware json schema from model

### ISSUE THREAD

**Issue by raratiru** [matthiask/django-translated-fields#27]
The idea is to use [django-ninja](https://django-ninja.rest-framework.com/) to automatically create a json schema from my model to feed [react-jsonschema-form](https://react-jsonschema-form.readthedocs.io/en/latest/).

The [line](https://github.com/vitalik/django-ninja/blob/v0.12.3/ninja/orm/fields.py#L108) that fetches the title is using `verbose_name`. I tried to override `verbose_name`:

```python
class Test(models.Model):
    point_name = TranslatedField(
        models.CharField(
            max_length=63,
            verbose_name=_("Point Name"),
        ),
        {"el": {"verbose_name": "Point Name (el)"}},
    )
```

With no luck as expected:

```python
from ninja.orm import create_schema
from my.models import Test

schema = create_schema(Test)
schema.schema()

>>> {'title': 'Test',
 'type': 'object',
 'properties': {'id': {'title': 'Id', 'type': 'integer'},
  'point_name_en': {'title': 'Point Name', 'maxLength': 63, 'type': 'string'},
  'point_name_el': {'title': 'Point Name', 'maxLength': 63, 'type': 'string'}}}
```
I can, of course do it manually, or use [djantic](https://jordaneremieff.github.io/djantic/usage/#customizing-the-schema) to override the relevant fields and set the title. 

Since all that seems to be a rather new technological mindset, related to tools like `pydantic`, `FastApi` and `SQLAlchemy`... I just wanted to say hallo wondering if there is something like a small change that could make the leap.

**Comment by matthiask** [comment]
We currently set `verbose_name` unconditionally here:
https://github.com/matthiask/django-translated-fields/blob/90856840a4a862ad5f61d711bc5f659cb3c461aa/translated_fields/fields.py#L103-L110

Maybe we could only create a `verbose_name` for language-specific fields when the `specific` dictionary doesn't contain a `verbose_name`?

Something like

```
field_kw = {**kwargs, **self.specific.get(language_code, {})}
field_kw.setdefault("verbose_name", _verbose_name_maybe_language_code(verbose_name, language_code)
f = self._field.__class__(*args, **field_kw)
```

Do you want to test this out and submit a pull request if it works?

**Comment by raratiru** [comment]
Trying to understand the code, I saw this fragment:

https://github.com/matthiask/django-translated-fields/blob/90856840a4a862ad5f61d711bc5f659cb3c461aa/translated_fields/fields.py#L34-L40

I am thinking that it would not be a good solution to provide multiple `verbose_name` fields because the content of the field is unique in all languages. 

I also understand that when a third party library programmatically fetches the verbose name, is not aware of how it is produced, it just asks for it.

What if we set as the default behavior to provide the language code and omit it only when certain conditions tackled by django-translated-fields are met?

**Comment by matthiask** [comment]
I think that if you specify a `verbose_name` for each language the specified values should be kept and not overridden. The `show_language_code` thing can be ignored in this case. `show_language_code` is only ever used when using the `TranslatedFieldAdmin` class (or by external code) so it should be fine to skip this.

I am against adding the language code to all `verbose_name` fields by default. The reason for this is that I do not want the language code to appear most of the time since I'm only showing values in the users' language anyway. The only exception to this is the CMS where content managers should be able to edit a field in all languages.

**Comment by matthiask** [comment]
@raratiru I think you found a real bug and I've fixed it in https://github.com/matthiask/django-translated-fields/commit/d3eaa9078bd964bf553204efa91449275cce7d6f . I have released 0.11.2 containing the fix.
